### ISSUE TITLE
Need documentation on usage 

### ISSUE THREAD

**Issue by rajkolli03** [csupnig/cointelligence#1]
@csupnig . I think it would be wonderful if there is any documentation or some readme on how to use and extend this particular code.

**Comment by csupnig** [comment]
You're absolutely right... unfortunately my time is pretty much blocked by other projects now. Would you be willing to start such a document?

**Comment by rajkolli03** [comment]
@csupnig . I can start such document. I am still trying to figure out the code given here and how this is fitting in cointelligence. Any pointers?

**Comment by csupnig** [comment]
This code is being deployed with cointelligence and is available as MathUtil in the strategies. It is not deployed automatically though... so your changes will only be reflected on the platform when it is updated. You can still use this code by adding it as a custom function in one of the strategies.
